.github/workflows/python-venv.yaml

```
@@ -0,0 +1,23 @@
+name: python
+on:
+  pull_request:
+jobs:
+  venv:
+    runs-on: ubuntu-latest
+    permissions: {}
+    container:
+      image: registry.fedoraproject.org/fedora:38
```

Hmm -- :latest ? I know it's more prone to "spontaneous" failures when Fedora updates, but (1) venv already isolates most of that, and (2) we won't forget to actually keep it up to date.

---

Sure.  Happy to let this fail on us in the future if something changes, rather than forgetting that it exists.